backend-plugin/src/main/java/com/redhat/jenkins/nodesharingbackend/ReservationTask.java

```diff
@@ -52,8 +56,17 @@
  *
  * @author ogondza.
  */
-public class ReservationTask extends AbstractQueueTask {
-    private static final Logger LOGGER = Logger.getLogger(ReservationTask.class.getName());
+public class ReservationTask extends AbstractQueueTask implements AccessControlled {
+    private static final Logger LOGGER = Logger.getLogger(com.redhat.jenkins.nodesharingbackend.ReservationTask.class.getName());
+
+    // TODO Should be better secured!
+    public ACL getACL() { return AuthorizationStrategy.Unsecured.UNSECURED.getRootACL(); }
```

Comment:
`Jenkins.getInstance().getACL()` so it respect the instance defaults. It should work for as the instance should be readable for anonymous.